Makefile

```diff
@@ -8,18 +8,16 @@ GOPATH:=$(shell go env GOPATH)
 OUTPUT_DIRNAME	?= output
 DOC_FILENAME	?= oci-image-spec
 
-PANDOC_CONTAINER ?= ghcr.io/opencontainers/pandoc:2.9.2.1-8.fc33.x86_64@sha256:5d81ff930a043295a557be8b003ece2a33d14e91b28c50d368413b83372f8d28
+PANDOC_CONTAINER ?= docker.io/pandoc/latex:3.1@sha256:1cf54d9214a9b52de2f58cf5895cc596a5960711a54d7938dc72f2b23473caf3
```

Comment:
Would be good to call out https://github.com/jgm/pandoc/issues/10952 for why we are pinned to this specific version just to make modifying this easier later on without having to reference the PR details.